src/index.html

```diff
@@ -11,8 +12,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;700&amp;family=Roboto:wght@400;500;700&amp;display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="container-card">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="container-card-image"
```

Comment:
```suggestion
          class="card__image"
```

Reply:
should such changes be applied to all classes that begin with container-...?

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/99878391/4e9064bc-cae3-4332-b183-88b4b3455a5f)
fixed it, but now I have the following errors